purchase_stock_picking_return_invoicing/README.rst

```diff
@@ -20,76 +20,66 @@ In the purchase order you are able to display, for each line:
+   *Delivered quantities*, and input some quantity to purchase.
 #. Confirm the purchase order.
 #. Go to *Shipment > Validate > Apply* so as to receive the quantities ordered.
-#. Press the button *Invoices* to create the vendor bill.
+#. Go back to the purchase order. Press the button *Vendor Bills* to create
+   the vendor bill.
 #. The proposed vendor bill will be proposed for the quantity received. The
-   *Invoice Status* is now 'Invoiced'
+   *Invoice Status* is now 'Invoice Received'
 #. Go to the original incoming shipment
-#. Press the button *Reverse*.
-#. In the wizard *Reverse Quantity* Set *Quantity* to the quantity to be
+#. Press the button *Return*.
+#. In the wizard *Reverse Transfer* set *Quantity* to the quantity to be
    returned. Press *Return* to complete the wizard.
 #. On the return picking press *Validate > Apply*.
 #. Go back to the purchase order. It will have  *Invoice Status* as 'Waiting
-   Invoces'. You will notice that the field *Returned Qty* is now the quantity
-   that was returned. The field *Quantity to Refund* is now the showing the
+   Invoices'. You will notice that the field *Returned Qty* is now the quantity
+   that was returned. The field *Qty to Refund* is now the showing the
    quantity returned that was previously billed.
-#. Press the button *Refunds* to create the vendor refund bill (since the
-   field *Quantity to Invoice* is negative).
+#. Press the button *Refunds* to create the vendor refund bill.
 #. The proposed vendor refund bill will be proposed for the quantity that is
    to be refunded.
 #. If you back to the purchase order, you will notice that *Invoice Status*
-   is now 'Invoiced', even when the quantity ordered does not match with the
-   quantity invoiced, because you did return some products.
+   is now 'Invoiced Received', even when the quantity ordered does not match
+   with the quantity invoiced, because you did return some products.
 
-Remark: As part of the 3-way match proces, if you accept that you will not
+Remark: As part of the 3-way match process, if you accept that you will not
 claim for a refund for the quantity returned to the supplier, just set the
 purchase status as 'Done' at the end of the process, and the quantity to
 invoice for the items will be set to 0 (because you have accepted the
 discrepancies).
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/95/9.0
-
-Known issues / Roadmap
-======================
-
-* The functionality of return processing with refunds is being discussed with
-  Odoo here: https://github.com/odoo/odoo/issues/13974, so this addon may not
-  need to be migrated to v10.
+   :target: https://runbot.odoo-community.org/runbot/95/10.0
 
-* The computation of the quantity invoiced is hacked to overcome an issue in
-  one of the tests of Odoo. See https://github.com/OCA/OCB/pull/598
```

Comment:
Yes, and the hack is removed as you can see here: https://github.com/OCA/account-invoicing/pull/283/commits/5097f62e495a5f192ee24df3b39ad61e180a31b9#diff-a03198f7ff7f7c9ef2454d443aff2b77L121.